tests/rbd/rbd_delete_protected_snapshot_krbd.py

```diff
@@ -0,0 +1,97 @@
+from tests.rbd.exceptions import RbdBaseException
+from tests.rbd.krbd_io_handler import krbd_io_handler
+from tests.rbd.rbd_utils import initial_rbd_config
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def rbd_delete_protect_snap(rbd, pool_type, **kw):
+    """
+    This module on krbd client - Performs deletion of protected snapshot on a RBD image
+    Args:
+        rbd: RBD object
+        pool_type: pool type (ec_pool_config or rep_pool_config)
+        **kw: test data
+    """
+    try:
+        pool = kw["config"][pool_type]["pool"]
+        image = kw["config"][pool_type]["image"]
+        snap = kw["config"][pool_type].get("snap", f"{pool}_{image}_snap")
+        clone_prefix = kw["config"][pool_type].get("clone", f"{pool}_{image}_clone")
+        size = kw["config"][pool_type]["size"]
+        snap_name = f"{pool}/{image}@{snap}"
+        image_spec = f"{pool}/{image}"
+
+        # Run fio and krbd map operations on image
+        kw.get("config")["image_spec"] = [image_spec]
+        kw.get("config")["size"] = size
+        kw["rbd_obj"] = rbd
+        krbd_io_handler(**kw)
```

Comment:
krbd_io_handler will again try to create a image in specified image_spec , we have provision to pass "do_not_create_image: True" in initial_rbd_config method, we can make use of that and avoid creating image while creating pool.

Reply:
Added the changed in the yaml under config.